deployment/src/main/java/io/quarkiverse/cxf/deployment/CxfWebServiceBuildItem.java

```diff
@@ -20,8 +19,14 @@
     private final String path;
     private final boolean isClient;
 
-    public CxfWebServiceBuildItem(String path, String sei, String soapBinding, String wsNamespace,
-            String wsName, List<String> classNames, String implementor) {
+    public CxfWebServiceBuildItem(
```

Comment:
Please remove all unnecessary formatting changes throughout the PR ... formatting changes like this make it quite difficult to tell at a later point whether something has really changed or not, especially when trying to identify a potential issue introduced in a PR.  Given the complexity of this PR, I would strongly recommend you fix this everywhere applicable.

Reply:
I see the problem: Disabled RedHat's Intellij Quarkus-Plugin because of issues. Now formatting is broken :-(

Reply:
To my best knowledge: fixed.